packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -313,7 +313,9 @@ export class OdspSummaryUploadManager {
                         currentPath,
                         expanded);
                     value = result.snapshotTree;
-                    unreferenced = summaryObject.unreferenced;
```

Comment:
do you want this configurable in some way, so you can test before having to ship new bits?

Reply:
Yes, Ideally this would be configurable and controlled by the runtime. I could not find a straightforward way to send this signal to the driver. 
I want to get this in the release today so this shorter change. I would ideally change runtime to not even send this flag to driver if not required (based on some configuration)

Reply:
https://github.com/microsoft/FluidFramework/blob/df6c53db9eccb288bc3b3f9393c80beef02a562a/packages/loader/driver-utils/src/blobAggregationStorage.ts#L31

Reply:
that function should be easy enough to parameterize

Reply:
Oh cool! I added a new function that does something similar. Didn't want to modify this one yet.